.gitlab/kitchen_testing/testing.yml

```
@@ -18,10 +18,10 @@
     - .kitchen_common
   after_script:
     - echo -n "--tags ci.job.name:${CI_JOB_NAME} --tags arch:${KITCHEN_ARCH} --tags os:${KITCHEN_PLATFORM} upload_option.os_version_from_name" > $CI_PROJECT_DIR/test/kitchen/tags.txt
-    - echo -n "${CI_JOB_URL}" > $CI_PROJECT_DIR/test/kitchen/job_url.txt
+    - echo -n "CI_JOB_URL=${CI_JOB_URL}" > $CI_PROJECT_DIR/test/kitchen/job_env.txt
```

```suggestion
    - echo -n "CI_JOB_URL='${CI_JOB_URL}'" > $CI_PROJECT_DIR/test/kitchen/job_env.txt
```
I'd feel more at ease if the value was quoted accordingly to avoid potential surprises